Clinical-Genomics/scout
# Show matching tiered variants

Show if one cancer variant has matching variants with Tier both on the variantS view and on the variant page. Fix #2680

![image](https://user-images.githubusercontent.com/28093618/120645633-eca09a80-c478-11eb-923b-a738bca6c9cb.png)

![image](https://user-images.githubusercontent.com/28093618/120645665-f62a0280-c478-11eb-966f-98e6ffc4b9c5.png)



**How to test**:
1. Create a duplicate load config file from cancer demo case config file (rename family name and case display name)
2. Load both cases.
3. Go to the of the 2 cancer cases and tag one variant with a tier
4. Go to the other variant and make sure that one variant is reported to have matching tiered variants on the cancer variantS page and on the variant page itself.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
